core/src/main/java/com/klaytn/caver/contract/ContractMethod.java

```diff
@@ -455,17 +518,40 @@ private void checkSendOption(SendOptions options) {
 
     private ContractMethod findMatchedInstance(List arguments) {
         // Check the parameter type defined in function and the parameter type passed are the same.
-        ContractMethod matchedMethod = null;
+        List<ContractMethod> matchedMethod = new ArrayList<>();
 
         if(this.getInputs().size() != arguments.size()) {
             for(ContractMethod method : this.getNextContractMethods()) {
                 if(method.getInputs().size() == arguments.size()) {
-                    matchedMethod = method;
-                    break;
+                    matchedMethod.add(method);
                 }
             }
         } else {
+            matchedMethod.add(this);
+        }
+
+        if(matchedMethod.size() == 0) {
+            throw new IllegalArgumentException("Cannot find method with passed parameters.");
+        }
+
+        if(matchedMethod.size() != 1) {
+            LOGGER.warn("It found a two or more overloaded function that has same parameter counts. It may be abnormally executed. Please use *withSolidityWrapper().");
```

Comment:
Printing this log in `call` or `send` function is better.
Otherwise, whenever call or send with solidity wrapper(for handling functions have the same number of parameters), this log will be printed.

Reply:
Oh, no never mind. I found `findMatchedInstanceWithSolidityWrapper` function is implemented.